disassemblers/ofrak_ghidra/ofrak_ghidra/server/__main__.py

```diff
@@ -7,18 +7,32 @@
     GHIDRA_START_SERVER_SCRIPT,
     GHIDRA_PATH,
     CORE_OFRAK_GHIDRA_SCRIPTS,
+    GHIDRA_USER,
+    GHIDRA_PASS,
+    GHIDRA_REPOSITORY_HOST,
+    GHIDRA_REPOSITORY_PORT,
 )
 
 
-def _run_ghidra_server(args):
+def _run_ghidra_server(*args):
     if sys.platform == "linux" or sys.platform == "darwin":
         subprocess.call(["chmod", "+x", GHIDRA_START_SERVER_SCRIPT])
```

Comment:
I know this isn't directly part of the PR, but I wonder if it makes sense to use the more portable [`os.chmod`](https://docs.python.org/3/library/os.html#os.chmod) here? More portable, but not necessarily more readable. 

```suggestion
        os.chmod(GHIDRA_START_SERVER_SCRIPT, os.stat(GHIDRA_START_SERVER_SCRIPT).st_mode | stat.S_IEXEC)
```